docs/tutorials/gluon/model_parallelism.md

```diff
@@ -0,0 +1,373 @@
+
+```
+
+
+```python
+train()
+```
+
+    [Epoch 1 Batch 100] loss 7.20, perplexity 1340.10
+    [Epoch 1 Batch 200] loss 6.66, perplexity 781.10
+    [Epoch 1 Batch 300] loss 6.65, perplexity 776.09
+    [Epoch 1 Batch 400] loss 6.55, perplexity 697.08
+    [Epoch 1 Batch 500] loss 6.46, perplexity 640.04
+    [Epoch 1 Batch 600] loss 6.38, perplexity 590.49
+    [Epoch 1 Batch 700] loss 6.36, perplexity 577.39
+    [Epoch 1 Batch 800] loss 6.22, perplexity 501.58
+    [Epoch 1 Batch 900] loss 6.10, perplexity 443.65
+    [Epoch 1 Batch 1000] loss 6.04, perplexity 418.13
+    [Epoch 1 Batch 1100] loss 6.15, perplexity 469.13
+    [Epoch 1 Batch 1200] loss 6.10, perplexity 446.42
+    [Epoch 1 Batch 1300] loss 6.08, perplexity 435.85
+    [Epoch 1 Batch 1400] loss 6.05, perplexity 424.83
+    [Epoch 1 Batch 1500] loss 6.01, perplexity 406.97
+    [Epoch 1 Batch 1600] loss 6.02, perplexity 410.54
+    [Epoch 1 Batch 1700] loss 5.98, perplexity 395.66
+    [Epoch 1 Batch 1800] loss 5.98, perplexity 396.62
+    [Epoch 1 Batch 1900] loss 5.81, perplexity 333.70
+    [Epoch 1 Batch 2000] loss 5.88, perplexity 356.13
+    [Epoch 1 Batch 2100] loss 5.96, perplexity 386.80
+    [Epoch 1 Batch 2200] loss 5.85, perplexity 347.08
+    [Epoch 1 Batch 2300] loss 5.78, perplexity 323.07
+    [Epoch 1 Batch 2400] loss 5.76, perplexity 318.46
+    [Epoch 1 Batch 2500] loss 5.74, perplexity 311.50
+    [Epoch 1 Batch 2600] loss 5.79, perplexity 328.09
+    [Epoch 1 Batch 2700] loss 5.79, perplexity 328.51
+    [Epoch 1 Batch 2800] loss 5.86, perplexity 350.40
+    [Epoch 1 Batch 2900] loss 5.73, perplexity 309.27
+    [Epoch 1 Batch 3000] loss 5.77, perplexity 319.66
+    [Epoch 1 Batch 3100] loss 5.65, perplexity 285.61
+    [Epoch 1 Batch 3200] loss 5.63, perplexity 279.74
+    [Epoch 1 Batch 3300] loss 5.62, perplexity 275.63
+    [Epoch 1 Batch 3400] loss 5.57, perplexity 263.28
+    [Epoch 1 Batch 3500] loss 5.59, perplexity 267.68
+    [Epoch 1 Batch 3600] loss 5.64, perplexity 281.89
+    [Epoch 1 Batch 3700] loss 5.71, perplexity 301.17
+    [Epoch 1 Batch 3800] loss 5.68, perplexity 293.86
+    [Epoch 1 Batch 3900] loss 5.69, perplexity 296.83
+    [Epoch 1 Batch 4000] loss 5.67, perplexity 289.74
+    [Epoch 1 Batch 4100] loss 5.48, perplexity 240.49
+    [Epoch 1 Batch 4200] loss 5.64, perplexity 281.97
+    [Epoch 1 Batch 4300] loss 5.61, perplexity 273.13
+    [Epoch 1 Batch 4400] loss 5.64, perplexity 281.21
+    [Epoch 1 Batch 4500] loss 5.65, perplexity 283.48
+    [Epoch 1 Batch 4600] loss 5.63, perplexity 278.12
+    [Epoch 1 Batch 4700] loss 5.68, perplexity 291.96
+    [Epoch 1 Batch 4800] loss 5.59, perplexity 267.57
+    [Epoch 1] time cost 267.24s, validation loss 5.63, validation perplexity 279.41
+
+
+
+```python
+test_L = eval(test_data)
+print('Best test loss %.2f, test perplexity %.2f'%(test_L, math.exp(test_L)))
+```
+
+    Best test loss 5.61, test perplexity 273.09
+
```

Comment:
you are missing the download notebook place holder

Reply:
Will add.